cluster/vagrant/provision-master.sh

```diff
@@ -34,6 +35,9 @@ if [[ "$(grep 'VERSION_ID' /etc/os-release)" =~ ^VERSION_ID=23 ]]; then
 fi
 
 NETWORK_IF_NAME=`echo ${if_to_edit} | awk -F- '{ print $3 }'`
+if [[ -z "$NETWORK_IF_NAME" ]]; then
+  NETWORK_IF_NAME="eth0"
```

Comment:
vSphere is not adding the 'VAGRANT-BEGIN' this causes the 'if_to_edit' to be empty, thus the 'NETWORK_IF_NAME' is also empty, which causes a wrong flannel configuration with empty network interface. As there is no simple way (that I know) to find the active network interface, I'm setting the NETWORK_IF_NAME to the standard 'eth0'